AutoRest/Generators/Java/Azure.Java.Tests/src/main/java/fixtures/paging/models/ProductResult.java

```diff
@@ -1,66 +0,0 @@
-/**
```

Comment:
@jianghaolu - When I executed gulp regenerate:expected, this Paging file got deleted. It seems right to me. However, it would be nice if you can confirm that this is fine.